php/OAuthSimple.php

```diff
@@ -490,9 +490,16 @@ private function _generateSignature ($parameters="")
 		{
             $secretKey .= self::_oauthEscape($this->_secrets['oauth_secret']);
         }
-        if(empty($parameters)){
-            $parameters = $this->_normalizedParameters();
+        
+        if(empty($parameters))
+        {
+            $parameters = urlencode($this->_normalizedParameters());
```

Comment:
Looking at the code, we may want to just swap rawurlencode with urlencode inside of _normalizedParameters(). We could also get rid of the ugly set of post encode fixes.